### PR TITLE
[WIP] Depends-On

### PR DESCRIPTION
**Pull request by grahamhayes**
Depends-On: kubic-project/salt#187

### CODE REVIEW

File: src/com/suse/kubic/Dependancies.groovy [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+		def addDependancy(String org, String repo, int num){
+
+			if (this.dependancies[org] == null) {
+				this.dependancies[org] = [:]
+			}
+
+			if (this.dependancies[org][repo] == null || this.dependancies[org][repo] == num) {
+				this.dependancies[org][repo] = num
+			} else {
+				throw new Exception("Repos cannot have 2 dependant PRs")
```
Comment (kiall) [inline src/com/suse/kubic/Dependancies.groovy]: Nit: error("Repos cannot have 2 dependant PRs") is the idiomatic way to do this with Jenkinsfile's

Comment (grahamhayes) [inline src/com/suse/kubic/Dependancies.groovy]: can that be done in a groovy class ?

Comment (kiall) [inline src/com/suse/kubic/Dependancies.groovy]: Oh - Maybe not.. Unsure.

**Comment by kiall** [comment]
Closing out, will revive when time allows.

**Pull request closed without merging**